src/components/forms/forms.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  getFormsDataThunk,
+  deleteFormSchemaThunk,
+  saveFormDataThunk
+} from './../../redux/forms-Reducer';
+import style from './forms.module.css';
+
+function Forms(props) {
+  const [state, setState] = useState({});
+
+  if (!props.checked) {
+    props.getFormsDataThunk();
+  } else if (!state.forms) setState({ forms: [...props.forms] });
+
+  const onSubmit = (e, key) => {
+    e.preventDefault();
+    let data = {};
+    const stateCopy = { ...state };
+    stateCopy.forms[key].inputs.forEach((item, index) => {
+      if (item.value)
+        data[(item.label + key + index).toLowerCase()] = item.value;
+      item.value = '';
+    });
+    props.saveFormDataThunk(data);
+    setState({ ...stateCopy });
+  };
+  const onHandelChnage = (e, key, index) => {
+    e.preventDefault();
+    const { value } = e.target;
+    const forms = state.forms;
+    forms[key].inputs[index].value = value;
+    setState({ forms });
+  };
+  const formsRender =
+    state.forms &&
+    state.forms.map((item, key) => {
+      return (
+        <div key={key} className={style.form_box}>
+          <div className={style.form_header}>
+            <button
+              style={{ backgroundColor: 'red' }}
+              onClick={() => {
+                props.deleteFormSchemaThunk(item._id);
+      
+              }}>
+              Delete
+            </button>
+            <span className={style.form_name}>{item.formName}</span>
+            <span className={style.form_desc}>{item.formDesc}</span>
+          </div>
+          <form
```

Comment:
Seems to me this component is so big, that you need to split it out, try to move form to a separate component,
remember don't go above 120 lines